apps/site/src/lib/components/Navbar/Navbar.module.scss

```diff
@@ -0,0 +1,58 @@
+.navMenuListWrapper {
+	// clip top only when mobile
+	@media (max-width: 768px) {
+		clip-path: inset(0px 0px -100px -100px);
+	}
+}
+
+.navMenuList {
+	@media (max-width: 768px) {
+		transform: translateY(-110%);
+		> * {
+			margin-bottom: 1rem;
+		}
+	}
+}
+
+// so the transition can be disabled on media break
+.transformTransition {
+	transition: transform 0.5s ease-in-out;
+}
+
+.hideList {
+	@media (max-width: 768px) {
+		transform: translateY(-110%);
+	}
+}
+
+.showList {
+	@media (max-width: 768px) {
+		transform: translateY(0px);
+	}
+}
+
+.navMenuLink {
+	$underline-size: 4px;
+
+	font-size: 24px;
+	text-shadow: 0px 0px 20px #fffce2;
+
+	// needed for underline effect to work
+	padding-bottom: 0.75rem;
+
+	background-image: linear-gradient(white, white);
+	background-repeat: no-repeat;
+	background-position-y: 35px;
+	background-size: 0% $underline-size;
```

Comment:
Suggestion [non-blocking]: given the lower weight of the sans-serif font, it may be more better to use a thinner underline effect.